charts/osm/values.yaml

```diff
@@ -83,3 +83,7 @@ OpenServiceMesh:
     # Destination's API or collector endpoint where the spans will
     # be sent to
     endpoint: "/api/v2/spans"
+
+  # Set to true to enable the osm-config validator webhook
+  # Currently disabled due to Issue #2264
+  enableConfigValidator: false
```

Comment:
Any reason to create a new value here vs. just commenting out the config temporarily to mirror c48acb9c61c8650a7ff0b4516de001c3d1728387? I don't see why anyone would ever set this to `true` while the issue gets fixed.

Reply:
Is there a Helm equivalent of `#if 0` ?

Reply:
You can use `{{/* a comment */}}` or I believe `{{ if false }}` would work too.

Reply:
Perfect, will try this.